src/telliot_feed_examples/reporters/interval.py

```diff
@@ -242,6 +244,10 @@ async def ensure_profitable(
                 gas_price = await ethgasstation(style=self.gas_price_speed)
                 self.legacy_gas_price = gas_price
 
+            if self.legacy_gas_price is None:
+                status.ok = False
+                return status
```

Comment:
Same about `error_status()`, and shouldn't this one also return a bool?

Reply:
added error_status(). The return type is ResponseStatus in this function.